libs/ng-multi-select-lib/src/lib/components/multi-select/multi-select.component.ts

```diff
@@ -131,4 +131,12 @@ export class MultiSelectComponent implements OnInit, ControlValueAccessor, After
     this.optionLabelsList = xor(this.optionLabelsList, [optionLabel]);
   }
 
+  @HostListener('document:click', ['$event.target'])
```

Comment:
Couldn't we find a way to set this host listener only when multi-select is opened ?